website/package.json

```diff
@@ -30,7 +30,7 @@
     "react-table": "^7.8.0"
   },
   "devDependencies": {
-    "@docusaurus/module-type-aliases": "^2.1.0"
+    "@docusaurus/module-type-aliases": "^2.2.0"
```

Comment:
could you also bump https://github.com/cloudposse/atmos/pull/253/files in package-lock.json in this pr too?